protoc-gen-grpc-gateway/gengateway/template.go

```diff
@@ -245,13 +253,13 @@ var (
 	{{- if and (eq (.HTTPMethod) "PATCH") (.FieldMaskField)}}
 	if protoReq.{{.FieldMaskField}} != nil && len(protoReq.{{.FieldMaskField}}.GetPaths()) > 0 {
 		runtime.CamelCaseFieldMask(protoReq.{{.FieldMaskField}})
-	} else {
-		if fieldMask, err := runtime.FieldMaskFromRequestBody(bytes.NewReader(body)); err != nil {
-			return nil, metadata, status.Errorf(codes.InvalidArgument, "%v", err)
-		} else {
-			protoReq.{{.FieldMaskField}} = fieldMask
-		}
-	}
+	} {{if not (eq "" .GetBodyFieldPath)}} else {
```

Comment:
I thought this was supposed to be `"*"`? Maybe I don't understand how `.GetBodyFieldPath` is supposed to work...